src/common/lib/utils/date.ts

```diff
@@ -32,3 +34,18 @@ export function localize(value: number, str: string): string {
 
   return `${value}${str}`;
 }
+
```

Comment:
We don't have to rewrite this since it' already here, but I think moment already just does this for us: https://momentjs.com/docs/#/displaying/fromnow/